src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,12 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={`TodoInfo ${todo.completed ? 'TodoInfo--completed' : ''}`}>
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    {todo.user ? (
+      <UserInfo user={todo.user} />
+    ) : ''}
```

Comment:
Consider it will be better
```suggestion
    {user && <UserInfo user={user} />}
```